src/main/java/org/terasology/thirst/ThirstAuthoritySystem.java

```diff
@@ -46,13 +56,27 @@ public void onPlayerSpawn(OnPlayerSpawnedEvent event, EntityRef player,
         player.saveComponent(thirst);
     }
 
+    /**
+     * Defines what to do when an entity is removed.
```

Comment:
This is a bit vague. You should describe what is happening/why this is important, e.g., "Updates the thirst value one last time before the entity is removed".